Dockerfile

```diff
@@ -8,20 +8,21 @@ WORKDIR /app/turtle
 RUN ./scripts/fetchRemoteTarballs.sh android
 
 RUN ln -s /app/turtle/workingdir /app/workingdir && \
-    for SDK_VERSION in `ls /app/workingdir/android/`; do \
-      echo "preparing $SDK_VERSION shell app" && \
-      cd /app/workingdir/android/$SDK_VERSION && \
-      mv packages non-workspace-packages && \
-      # Keep in sync with src/bin/setup/android/index.ts#_installNodeModules
-      # --prod requires --ignore-scripts (otherwise
-      # expo-yarn-workspaces errors as missing)
-      yarn install --ignore-scripts --prod && \
-      mv non-workspace-packages packages ; \
-    done
+  for SDK_VERSION in `ls /app/workingdir/android/`; do \
+  echo "preparing $SDK_VERSION shell app" && \
+  cd /app/workingdir/android/$SDK_VERSION && \
+  mv packages non-workspace-packages && \
+  # Keep in sync with src/bin/setup/android/index.ts#_installNodeModules
+  # --prod requires --ignore-scripts (otherwise
+  # expo-yarn-workspaces errors as missing)
+  yarn install --ignore-scripts --prod && \
+  mv non-workspace-packages packages ; \
+  done
```

Comment:
```suggestion
    for SDK_VERSION in `ls /app/workingdir/android/`; do \
      echo "preparing $SDK_VERSION shell app" && \
      cd /app/workingdir/android/$SDK_VERSION && \
      mv packages non-workspace-packages && \
      # Keep in sync with src/bin/setup/android/index.ts#_installNodeModules
      # --prod requires --ignore-scripts (otherwise
      # expo-yarn-workspaces errors as missing)
      yarn install --ignore-scripts --prod && \
      mv non-workspace-packages packages ; \
    done
```